models/sessions/scratch/default/snowplow_web_sessions_this_run.sql

```diff
@@ -192,6 +229,21 @@ select
     -- engagement fields
     c.page_views,
     c.engaged_time_in_s,
+    {%- if var('snowplow__list_event_counts', false) %}
+        {% if target.type in ['postgres'] %}
+            cast(event_counts_string as json) as event_counts,
+        {% elif target.type in ['redshift'] %}
+            JSON_PARSE(event_counts_string) as event_counts,
+        {% endif %}
+    {% endif %}
+    c.total_events,
+        c.page_views >= 2 or c.engaged_time_in_s / {{ var('snowplow__heartbeat', 5) }} >= 2
```

Comment:
Again feels like we're hardcoding specific engagement logic which may not apply to the individual